launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -433,6 +439,36 @@ QProcessEnvironment MinecraftInstance::createEnvironment()
     return env;
 }
 
+QProcessEnvironment MinecraftInstance::createLaunchEnvironment()
+{
+    // prepare the process environment
+    QProcessEnvironment env = createEnvironment();
+
+#ifdef Q_OS_LINUX
+    if (settings()->get("EnableMangoHud").toBool())
+    {
+        auto preload = env.value("LD_PRELOAD", "") + ":libMangoHud_dlsym.so:libMangoHud.so";
+        auto lib_path = env.value("LD_LIBRARY_PATH", "") +  ":/usr/local/$LIB/:/usr/local/$LIB/mangohud/:/usr/$LIB/mangohud/";
```

Comment:
There can be other stuff in `/usr/local/lib` that might cause unexpected issues. If the user wants us to load stuff from there, they should define the necessary env vars themselves.

```suggestion
        auto lib_path = env.value("LD_LIBRARY_PATH", "") +  ":/usr/local/$LIB/mangohud/:/usr/$LIB/mangohud/";
```